components/CreateEventForm/CreateEventForm.js

```diff
@@ -1,51 +1,120 @@
-import React from 'react';
-import { StyleSheet, View, Button, Text, TextInput} from 'react-native';
+import React, { useState } from 'react';
+import ModalDropdown from 'react-native-modal-dropdown';
+import { StyleSheet, View, Button, Text, TextInput, Alert } from 'react-native';
 
 export default function CreateEventForm() {
+
+  const [ nameOfSport, setNameOfSport ] = useState('')
+
+  const validateInputs = () => {
+    Alert.alert(nameOfSport)
+  }
+
+  const handleNameOfSportChange = e => {
+  setNameOfSport(e.target.value);
+  }
+
+  //  ** Name of Sport should be a dropdown
+
   return (
     <View style={styles.container}>
-      <View style={styles.closeButton}><Button title='X'/></View>
-      <Text style={styles.text}>CREATE A NEW EVENT FORM!</Text>
-      <TextInput style={styles.input} placeholder='Event Name...'/>
+      <Text style={styles.text}>CREATE A NEW EVENT</Text>
+      <TextInput
+        style={styles.input}
+        placeholder='Name of Sport...'
+        onChangeText={handleNameOfSportChange}
+        value={nameOfSport}
+      />
```

Comment:
I'll definitely need to use this input code when implementing the filter events functionality